delphi/translators/for2py/pyTranslate.py

```diff
@@ -1127,24 +1169,24 @@ def get_range(self, node):
         up = node["high"]
         # Get lower bound value
         if low[0]["tag"] == "literal":
-            loBound = self.proc_literal(low[0])
+            loBound = self.proc_literal(low[0], printState)
```

Comment:
Elsewhere in this file we use underscores in variable names.  To be consistent, this should be `lo_bound` rather than `loBound`.